docs/shared-set-up.asciidoc

```diff
@@ -0,0 +1,148 @@
+// Content in this file is shared across the Node.js documentation.
```

Comment:
@Qard -- This is for https://github.com/elastic/apm-agent-nodejs/issues/1023. All six of the getting started pages share a ton of content. The idea here is to move the shared content into one file and then include it in each of the getting started files. This adds a bit of complexity to the docs, but also makes it easier to update them if things change.

Curious to hear your thoughts on this approach. If you think it makes it too complicated, we can chat about alternative solutions.

Reply:
Seems straightforward enough. Anything that makes it _easier_ to maintain docs while keeping a high _quality_ standard is a good thing. 😸

Reply:
Great!